gamemodes/terrortown/entities/weapons/weapon_tttbase.lua

```diff
@@ -162,9 +162,9 @@ SWEP.ReloadAnim = ACT_VM_RELOAD
 SWEP.fingerprints = {}
 
 -- Time needed to enter / leave the ironsight in seconds
-SWEP.IronSightTime = 0.25
+SWEP.IronSightsTime = 0.25
 -- The position offset applied when entering the ironsight
-SWEP.IronSightPos = Vector(0, 0, 0)
+SWEP.IronSightsPos = Vector(0, 0, 0)
 -- The rotational offset applied when entering the ironsight
 SWEP.IronSightsAng = Vector(0, 0, 0)
```

Comment:
After looking at git blame, i think we should remove all those values.

They were introduced in https://github.com/TTT-2/TTT2/pull/456
Also they are not present in normal TTT and i cannot see where it is used in TTT2.

And additionally these would clash with the network var values. eg. GetIronSightstime() should be used instead i think.

Reply:
Indeed, youre correct for the time.
But i still think we should remove all of them, since i don't think it is sensible to assume 0-vectors to be correct there.
Especially since the code behaves differently when those are not set. (And all calls check if those values even exist).

Reply:
Removal from just `weapon_tttbase`, or for all SWEPs? Just so I'm not misunderstanding.

Reply:
Base as before and as in normal TTT but i want to hear someone else about this

Reply:
Wasnt sure before, but as TTT also doesnt set it, I'd say we just comment it out. So we have the same missing zoom behaviour in TTT2 as in TTT.
So also remove SWEP.IronSightsAng

And definitely remove the sightsTime. 👍

Reply:
Roger